packages/server/src/modules/user/index.ts

```diff
@@ -3,12 +3,14 @@ import { getUser as getUserMiddleware } from './middleware';
 import {
   addUser as addUserService,
   getUser as getUserService,
+  getPublicCoaches as getPublicCoachesService,
 } from './service';
 
 import './routes';
 
 application.middleware.getUser = getUserMiddleware;
 application.service.getUser = getUserService;
 application.service.addUser = addUserService;
+application.service.getPublicCoaches = getPublicCoachesService;
```

Comment:
You don't need to add this service to the application service since it's not used outside the module.